src/bugs/create.ts

```diff
@@ -23,15 +23,17 @@ export = function (Bugs: BugsInterface) {
     Bugs.create = async function (data : Bug): Promise<number> {
         const timestamp = data.timestamp || Date.now();
```

Comment:
Same question about this timestamp too.

Reply:
I still technically need `Date.now()` for adding to the sorted set if we were ever to fetch bugs from the db. It is not an actual field anymore though, you are rightt